pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -27,6 +27,7 @@ type Retriever struct {
 	plan          *Plan                        // plan instance
 	volumeKeys    []string                     // list of keys found
 	bindingPrefix string                       // prefix for variable names
+	Cache         map[string]interface{}	// store visited paths
```

Comment:
Can this be `cache` instead of `Cache`?

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/159/commits/601169cda0eefd1de21c503f001e38d1f104d626